src/v/cluster/health_monitor_frontend.cc

```diff
@@ -84,4 +108,45 @@ health_monitor_frontend::get_cluster_health_overview(
         return be.get_cluster_health_overview(deadline);
     });
 }
+
+void health_monitor_frontend::set_cluster_disk_health(
+  const storage::disk_space_alert d) {
+    _cluster_disk_health = d;
+}
+
+ss::future<> health_monitor_frontend::broadcast_disk_health(
+  const storage::disk_space_alert dsa) {
+    co_await container().invoke_on_others(
```

Comment:
why not `invoke_on_all`?

Reply:
No reason--other than I wasn't sure it was allowed based on reading the [description](http://docs.seastar.io/master/classseastar_1_1peering__sharded__service.html). "allows access to peers" maybe could be written "allows access to all shards" if that is the case.

Reply:
Is it a bug though? Because then the calling shard never updates the state?

Reply:
The caller sets locally and then broadcasts to others. Maybe that function name could be `update_other_shards` instead of `broadcast`

Reply:
I changed the names and eliminated the unnecessary setter on the latest patches. LMK what you think.